system/include/emscripten/wasmfs.h

```diff
@@ -57,7 +57,18 @@ backend_t wasmfs_create_memory_backend(void);
 //
 // TODO: Add an async version of this function that will work on the main
 // thread.
-backend_t wasmfs_create_fetch_backend(const char* base_url __attribute__((nonnull)));
+backend_t wasmfs_create_fetch_backend(const char* base_url __attribute__((nonnull)),
+                                      uint32_t chunkSize,
+                                      void *manifest);
+
+// Create a FetchFS manifest record that can be populated and passed into
+// wasmfs_fetch_create_backend.
+void *wasmfs_fetch_create_manifest();
```

Comment:
A manifest is a std::map<string,string> of path to URL. I think there needs to be wasmfs documentation in general, and if there’s an rst file where the usage documentation should go I’d be happy to write up docs for fetchfs once the API sketch here is approved.

Reply:
Hmm, the best we have for WasmFS docs for now is `site/source/docs/api_reference/Filesystem-API.rst`, but it's very minimal.

I think we can add some explanation in this header file though. It would be nice if it were self-contained so people could understand it without other docs. Though I do agree we should have those other docs too.